wordpress-mobile/WordPress-iOS
# Improves accessibility labels and hints in media library

Fixes [#12874](https://github.com/wordpress-mobile/WordPress-iOS/issues/12874)

To test:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
2. In the WordPress app, go to My Site > Media.
3. Explore the buttons in the top bar for add, edit, and trash (after you go into edit mode).  VoiceOver should now be more descriptive for the add and edit buttons, and the trash button should now be accessible


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
I tested in Simulator using Accessibility Inspector (I currently don't have ability to test / debug on Device)

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
